capa/features/extractors/dnfile/extractor.py

```diff
@@ -98,7 +99,7 @@ def get_functions(self) -> Iterator[FunctionHandle]:
             fh: FunctionHandle = FunctionHandle(
                 address=DNTokenAddress(token),
                 inner=method,
-                ctx={"pe": self.pe, "calls_from": set(), "calls_to": set(), "cache": self.token_cache},
+                ctx={"pe": self.pe, "calls_from": set(), "calls_to": set(), "cache": self.token_cache, "len": self.len},
```

Comment:
update this name, too, please.

Reply:
incidentally, this is a great way to pass the string length configuration to the extractors. could we use this strategy more consistently to avoid having to update all the `extract_*` callback function signatures?

Reply:
I created a `file_ctx` dict to pass into `extract_file_features` functions. This decreases the number of kwargs that need to be added to the function signatures.